charts/rancher-turtles/values.yaml

```diff
@@ -25,6 +20,7 @@ rancherTurtles:
       enabled: true
     agent-tls-mode:
       enabled: false
+    kubectlImage: registry.k8s.io/kubernetes/kubectl:v1.30.0
```

Comment:
maybe this can be moved to the top level along with the CAPI operator one? so users set it only in one place